python_modules/dagster/dagster/_core/definitions/decorators/assets_definition_factory.py

```diff
@@ -0,0 +1,52 @@
+from functools import cached_property
```

Comment:
should this file be named `in_out_mapper.py`?

Reply:
`InOutMapper` gets renamed upstack.